client/package.json

```diff
@@ -6,19 +6,17 @@
   "dependencies": {
     "@blockstack/connect": "^2.3.1",
     "@fortawesome/free-brands-svg-icons": "^5.12.1",
-    "@material-ui/core": "^4.9.7",
```

Comment:
For some reason, looks like this got removed / the `package.json` got switched up which is why build is failing

Reply:
Oops, my bad. I installed @material-ui core at another branch, I thought it wasn't being used anywhere so I removed it. I'll reinstall it as a dependency and then re-push another branch.

Reply:
No prob! Looks like now this just created a merge conflict on your PR - you'll just have to rebase off the current master to fix that.

[Here's a guide to point you in the right direction](https://stackoverflow.com/questions/15602037/git-rebase-upstream-master-vs-git-pull-rebase-upstream-master) - feel free to lmk if you have any questions!